blocks/adp-collection-header/adp-collection-header.js

```diff
@@ -36,7 +55,7 @@ function createCollectionInfoHeader(collectionInfoHeader, collection) {
         'Delete collection',
         `Are you sure you want to delete the collection "${collection.title}"?`,
         async () => {
-          await deleteCollection(collectionId, collection.title);
+          await deleteCollection(collectionId, collection.title, collection.etag);
```

Comment:
@TyroneAEM Is this code `collection.etag` needed in `Select All `Task PR?

Reply:
Good catch, the etag should not be in this code.
I have changed it back to `await deleteCollection(collectionId, collection.title);`

I am not sure how this got changed as I branched off the main branch, the
`await deleteCollection(collectionId, collection.title, collection.etag);`
is the new code from Collections API branch.